internal/config/application.go

```diff
@@ -150,7 +152,7 @@ func (cfg *Application) parseLogLevelOption() error {
 		case v >= 2:
 			cfg.Log.LevelOpt = logrus.DebugLevel
 		default:
-			cfg.Log.LevelOpt = logrus.ErrorLevel
+			cfg.Log.LevelOpt = logrus.WarnLevel
```

Comment:
The PR description doesn't mention this default log level change, should we default to warning in syft too?

Reply:
The problem is by default warning messages do not show up, but I do not believe the messages I want a user to see are errors... it's a case where a user _should_ see something, but it hasn't necessarily prevented the process from running. I think logging warn and higher messages is fairly typical.

Reply:
What about using the TUI to present that information?

Since this is the global log setting, all other warning messages will be shown, which might be a problem.